packages/eas-cli/src/credentials/ios/utils/printCredentialsBeta.ts

```diff
@@ -0,0 +1,93 @@
+import chalk from 'chalk';
+import dateformat from 'dateformat';
+
+import { CommonIosAppCredentialsFragment, IosDistributionType } from '../../../graphql/generated';
+import log from '../../../log';
+import { fromNow } from '../../../utils/date';
+import { AppLookupParams } from '../api/GraphqlClient';
+
+export function prettyIosDistributionType(distributionType: IosDistributionType): string {
+  switch (distributionType) {
+    case IosDistributionType.AppStore:
+      return 'App Store';
+    case IosDistributionType.AdHoc:
+      return 'Ad Hoc';
+    case IosDistributionType.Development:
+      return 'Development';
+    case IosDistributionType.Enterprise:
+      return 'Enterprise';
+    default:
+      return 'Unknown';
+  }
+}
+
+export function displayEmptyIosCredentials(appLookupParams: AppLookupParams): void {
+  const { projectName, bundleIdentifier } = appLookupParams;
+  log(chalk.bold(`iOS Credentials`));
+  log(`  Project: ${projectName}`);
+  log(`  Bundle Identifier: ${bundleIdentifier}`);
+  log(`  No credentials set up yet!`);
+}
+
+export function displayAllIosCredentials(
```

Comment:
This function displays credentials for one app and for one specific built type, using `All` in the name is confusing here.